### PR TITLE
Add support for firewalld

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

**Comment by rpelisse** [comment]
@guidograzioli whenever you have the time, please review!

File: roles/wildfly_firewalld/meta/argument_specs.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+                default: "false"
+                description: "Indicate if firewalld is enabled for this collection"
+                type: "str"
+            wildfly_listen_ports:
+                default:
+                  - 8009
+                  - 8080
+                  - 8443
+                  - 4712
+                  - 4713
```
Comment (guidograzioli) [inline roles/wildfly_firewalld/meta/argument_specs.yml:18]: Can you add 7600 which is the default jgroups-tcp port?

Comment (rpelisse) [reply roles/wildfly_firewalld/meta/argument_specs.yml:18]: The idea would be that if you don't use standalone, but standalone_full.xml, you'll need to provide the ports.

Maybe we can have a standalone_ports: var and a standalone_full_ports: with some sort of switch / if to select the right right set of ports depending on the config?

Thoughts?

Comment (guidograzioli) [reply roles/wildfly_firewalld/meta/argument_specs.yml:18]: no it's ok; i didn't notice the jgroup-tcp socket binding is only defined in standalone-ha!

**Pull request merged**